.gitignore

```diff
@@ -11,6 +11,7 @@ node_modules/
 local/
 quicktests/github_token.txt
 quicktests/overlaying/list_of_quicktests.json
+*.log
```

Comment:
`npm-debug.log`, but also there aren't any .logs we should add, so just using the wildcard